qemu/tests/cfg/scsi_mq_check.cfg

```diff
@@ -0,0 +1,14 @@
+- scsi_mq_check:
+    # As WinXP, Win2003, Win2008, Win7 not support multi queue. So don't test them.
+    no WinXP Win2003 Win2008 Win7
+    no RHEL.4 RHEL.5 RHEL.6
+    only virtio_scsi
+    type = scsi_mq_check
+    driver_name = vioscsi
+    # for linux:
+    pattern = "PCI-MSI-edge\s+virtio0-request"
```

Comment:
is there virtio1,virtio2.....? I'm not sure if "virtio0-request" is the fixed string.